src/input/pipewire.c

```diff
@@ -18,8 +18,8 @@ static void on_process(void *userdata) {
     struct pw_data *data = userdata;
     struct pw_buffer *b;
     struct spa_buffer *buf;
+    float *samples;
     uint32_t n_samples;
-    int16_t *samples;
 
```

Comment:
Again, looks like these changes I also didn't notice during refactoring. Possible need to rollback once PR is approved to merge